newton/_src/utils/import_mjcf.py

```
@@ -841,6 +850,64 @@ def parse_shapes(
+                    if "fromto" in geom_attrib:
+                        ft = np.fromstring(geom_attrib["fromto"], sep=" ")
+                        h_full = np.linalg.norm(ft[:3] - ft[3:]) * scale
+                        half_height = h_full * 0.5
+                    else:
+                        half_height = geom_size[1]  # already half-height
+                    m_computed, c, I = compute_inertia_capsule(1.0, r, half_height)
+                    if m_computed > 1e-6:
+                        I = I * (geom_mass_explicit / m_computed)
+                elif geom_type == "ellipsoid":
+                    a, b, c = geom_size[0], geom_size[1], geom_size[2]
+                    m_computed, c, I = compute_inertia_ellipsoid(1.0, a, b, c)
```

Ruff (0.15.2)

[error] Ambiguous variable name: `I`

(E741)